src/types/Todo.ts

```diff
@@ -4,3 +4,8 @@ export interface Todo {
   completed: boolean;
   userId: number;
 }
+
+export type InitialStates = {
```

Comment:
This is a not ok name, especially for the type

Please rename it (Filter, TodoFilter, TodoFilterOptions, etc)